.rubocop.yml

```diff
@@ -121,6 +121,8 @@ Style/PredicateName:
 Style/GuardClause:
   Exclude:
     - 'lib/active_fedora/autosave_association.rb'
+    - 'lib/active_fedora/file_configurator.rb'
```

Comment:
@mjgiarlo IIRC, it wanted me to put a guard clause here:
https://github.com/projecthydra/active_fedora/blob/master/lib/active_fedora/file_configurator.rb#L151:L162
at line 159, which I think was its veiled way of saying you have too many `elsif`s. I don't really have the time to pick it apart, but we could ticket it if you like.